app/models/jurisdiction.rb

```diff
@@ -130,7 +171,9 @@ def search_data
       permit_applications_size: permit_applications_size,
       user_ids: users.pluck(:id),
       submission_inbox_set_up: submission_inbox_set_up,
-      created_at: created_at
+      created_at: created_at,
+      review_manager_emails: review_manager_emails,
+      regional_review_manager_emails: regional_review_manager_emails
```

Comment:
This might be better if they were the same field, like manager_emails or something